docs/authentication/stytch/permit-integration.mdx

```diff
@@ -0,0 +1,406 @@
+---
+sidebar_position: 1
+title: Stytch & Permit Integration
+---
+
+Stytch is one of the leading authentication platforms for developers, offering robust and scalable authentication solutions.
+Permit.io is a leading authorization platform, providing easily manageable and secure authorization capabilities. By integrating
+Stytch with Permit.io, you can enhance your application with both strong authentication and dynamic authorization.
+
+In this guide, you'll learn how to add Stytch-powered authentication to application using their **frontend JavaScript SDK**
+and their **backend Node SDK**, along with their pre-built login UI component.
+
+---
+
+:::note
+There are many different ways to implement Stytch, and the specific strategy you choose will depend on your use case and preferences.
+This guide is intended to demonstrate one way of implementing Stytch in order to get you up and running as quickly as possible.
+
+We will base our examples on working with a Next.js application. However, Stytch provides example applications in **various programming
+languages** to suit different development needs.
+:::
+
+## What you will build
+
+In this tutorial, you’ll learn how to integrate Stytch for secure authentication and Permit.io for easily manageable authorization,
+allowing us to change our application's permissions on the fly.
+
+:::note
+This guide provides a basic introduction to integrating Stytch with Permit.io for authentication and authorization.
+**The authorization functionality demonstrated will be quite simple.**
+:::
+
+## Prerequisites
+
+- Stytch account with a Stytch Consumer project via the Stytch Dashboard. You can read more about doing
+  so [here](https://stytch.com/docs/guides/quickstarts/nextjs#before-you-start).
+- Permit account
+- Basic understanding of Next.js App Router
```

Comment:
Maybe I'm missing something, but I'm not sure how's the App Router relevant to the tutorial. Are we using it anywhere?